src/notification.rs

```diff
@@ -330,6 +343,44 @@ impl Notification {
         Ok(NotificationHandle::new(self.clone()))
     }
 
+     /// Sends Notification to NSUserNotificationCenter.
+    ///
+    /// Returns an `Ok` no matter what, since there is currently no way of telling the success of
+    /// the notification.
+    #[cfg(target_os = "windows")]
+    pub fn show(&self) -> Result<()> {
+        let sound_name = self.sound_name.clone();
+        let sound = match sound_name {
+            Some(chosen_sound_name) => winrt_notification::Sound::from_str(&chosen_sound_name).ok(),
+            None => None
+        };
+
+        let duration = match self.timeout {
+            Timeout::Default => winrt_notification::Duration::Short,
+            Timeout::Never => winrt_notification::Duration::Long,
+            Timeout::Milliseconds(t) => if t >= 25000 {
+                winrt_notification::Duration::Long
+            } else {
+                winrt_notification::Duration::Short
+            }
+        };
+
+        let mut toast = Toast::new(Toast::POWERSHELL_APP_ID) //Not using app name due winrt-notification#1
+            .title(&self.summary)
+            .text1(&self.subtitle.as_ref().map(|s| &**s).unwrap_or("")) // subtitle
```

Comment:
this might not be much better, but perhaps a little less intimidating
```suggestion
            .text1(&self.subtitle.as_ref().map(AsRef::as_ref).unwrap_or("")) // subtitle
```